addons/product/views/product_views.xml

```diff
@@ -437,22 +437,26 @@
                     <progressbar field="activity_state" colors='{"planned": "success", "today": "warning", "overdue": "danger"}'/>
                     <templates>
                         <t t-name="kanban-box">
-                            <div class="oe_kanban_global_click">
-                                <div class="o_kanban_image">
+                            <div class="oe_kanban_card oe_kanban_global_click">
+                                <div class="o_kanban_image me-1">
                                     <img t-att-src="kanban_image('product.product', 'image_128', record.id.raw_value)" alt="Product" class="o_image_64_contain"/>
                                 </div>
                                 <div class="oe_kanban_details">
-                                    <field name="priority" widget="priority" readonly="1"/>
-                                    <strong class="o_kanban_record_title">
-                                        <field name="name"/>
-                                        <small t-if="record.default_code.value">[<field name="default_code"/>]</small>
-                                    </strong>
+                                    <div class="o_kanban_record_top mb-0">
+                                        <div class="o_kanban_record_headings">
+                                            <strong class="o_kanban_record_title">
+                                                <field name="name"/>
+                                            </strong>
+                                        </div>
+                                        <field name="priority" widget="priority"/>
```

Comment:
the previous `priority` was `readonly="1"`, though I'm not sure why.
Is there a stated reason for this change though?